src/test/java/com/google/jenkins/plugins/k8sengine/KubernetesEngineBuilderPipelineIT.java

```diff
@@ -99,6 +99,7 @@ public static void init() throws Exception {
     envVars.put("CLUSTER_NAME", clusterName);
     envVars.put("CREDENTIALS_ID", credentialsId);
     envVars.put("ZONE", testZone);
+    envVars.put("NAMESPACE", "default");
```

Comment:
We should also include a test specifying a none default namespace, that way we can ensure we're handling a case similar to the issue which surfaced in the latest friction log.